src/dev/i18n/extractors/code.test.js

```diff
@@ -46,6 +46,42 @@ class Component extends PureComponent {
 }
 `);
 
+const extractCodeMessagesSource2 = Buffer.from(`
+i18n('kbn.mgmt.id-1', { defaultMessage: 'Message text 1' });
```

Comment:
is this `i18n` needed here?

Reply:
I was following the previous example from here:
https://github.com/elastic/kibana/blob/b01bbbf3447b1c07ab23f8dbb33ea8feb7a3b18f/src/dev/i18n/extractors/code.test.js#L31

Wanted to check if the values will still be extracted if there are multiple strings in a test.